site/src/component/SideInfo/SideInfo.tsx

```diff
@@ -161,6 +162,10 @@ const SideInfo: FC<SideInfoProps> = (props) => {
         </div>
       </div>
 
+      <div className="side-info-recent-offerings">
+        <RecentOfferings terms={props.terms ? props.terms : []} />
```

Comment:
I don't think you need to wrap this in a `div` – you already have a wrapper container inside it. Instead, just use the one that's there already, change the `className` to `recent-offerings`, and combine the styles